doc/intro.rst

```diff
@@ -26,12 +26,18 @@ colors) with a switchable backlight:
 
 .. image:: images/st7735.jpg
 
+The HT1621 display (as purchased) supports six 7-segment characters with a 
```

Comment:
not sure why the `(as purchased)` part is in there.

My ordered display also arrived and I'll test this asap.